be/src/runtime/runtime_predicate.cpp

```diff
@@ -161,6 +156,10 @@ Status RuntimePredicate::update(const Field& value, const String& col_name, bool
         return Status::OK();
     }
 
+    // TODO defensive code
+    if (!_tablet_schema || !_tablet_schema->have_column(col_name)) {
```

Comment:
when will `!_tablet_schema->have_column(col_name)` happened? maybe we could add some comment?